x/mint/types/params.go

```diff
@@ -50,7 +50,7 @@ func NewParams(
 func DefaultParams() Params {
 	return Params{
 		MintDenom:               sdk.DefaultBondDenom,
-		GenesisEpochProvisions:  sdk.NewDec(6_700_000).QuoInt64(365),
+		GenesisEpochProvisions:  sdk.NewDec(6_700_000_000_000),
```

Comment:
let's revert this, I think I made this change by mistake while trying to make a change to osmosis' mint params
```suggestion
		GenesisEpochProvisions:  sdk.NewDec(6_700_000).QuoInt64(365),
```

Reply:
done